Good-Bids/goodbids
# [#495] Setting up a default Navigation

# Summary
The option to render inner blocks inside of core/navigation will be in WP version 6.5. So this is a workaround for now. See the notes in the ticket for more details #495.

This updates the `wp_navigation` content with the two About and Auctions page using ``. This allows us (and the user) to have full control of the navigation block in Gutenberg. 

- Updates the options with the About and Auction page id on site creation. 
- Navigation runs on Site verification.
- Updates the navigation content.

### Things still TODO:
- Figure out why it does not fire on setup.
- ~~Figure out how to get nav ID - it is always the first one.~~

## Issues

* #495 

## Testing Instructions

1. Set up a new site and fill out all the information to verify the site.
2. Go to the front-end of the site and make sure the navigation is updated. 
3. Edit the navigation and make sure you can add more or update the current links. 

## Screenshots

### Front-end
<img width="586" alt="Screenshot 2024-03-01 at 3 56 52 PM" src="https://github.com/Good-Bids/goodbids/assets/91974372/9a4158f1-f9f6-4d52-bb48-3d4f2ef41ca5">


### Admin
<img width="1594" alt="Screenshot 2024-03-01 at 3 57 33 PM" src="https://github.com/Good-Bids/goodbids/assets/91974372/3b51c2ea-e222-4ce0-82fb-492f46fc289c">
